src/index.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './style/index.css';
+import { Provider } from 'react-redux';
 import App from './App';
+import store from './redux/store';
 
 ReactDOM.render(
   <React.StrictMode>
+    <Provider store={store} />
```

Comment:
I though this `Provider` needed to wrap `App`, but it's good to see that it works this way too!